### PR TITLE
feat(kt-filters): support data-test attr

### PR DESCRIPTION
**Pull request by santiagoballadares**

### CODE REVIEW

File: packages/kotti-ui/source/kotti-filters/components/FilterActions.vue [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 	<div class="kt-filter__actions">
 		<ButtonLink
+			:data-test="dataTest ? `${dataTest}-add-filter` : null"
 			:isDisabled="isAddDisabled"
 			:isLoading="isLoading"
 			:label="translations.addFilterLabel"
 			@click="handleAdd"
 		/>
 		<ButtonLink
+			:data-test="dataTest ? `${dataTest}-clear-all` : null"
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-filters/components/FilterActions.vue:11]: Should either be `:dataTest=` or `: undefined` depending on if this is intended to be passed as a prop or as a html attribute

File: packages/kotti-ui/source/kotti-filters/KtFilters.vue [hunk 2]
```diff
@@ -3,12 +3,14 @@
...
 			v-if="hasSearchColumn"
 			:column="searchColumn"
+			:dataTest="dataTest"
 			:filter="searchValue"
 			:isLoading="isLoading"
 			@input="setSearchFilter"
 		/>
 		<div ref="listTriggerRef">
 			<ButtonLink
+				:data-test="dataTest ? `${dataTest}-show-filters` : null"
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-filters/KtFilters.vue:13]: Should either be `:dataTest=` or `: undefined` depending on if this is intended to be passed as a prop or as a html attribute

File: packages/documentation/pages/usage/components/filters.vue [hunk 3]
```diff
@@ -8,6 +8,7 @@
     			<KtFilters
     				v-model="filters"
     				:columns="componentProps.columns"
+    				dataTest="kt-filters"
```
Comment (FlorianWendelborn) [inline packages/documentation/pages/usage/components/filters.vue:11]: Won’t work with more than 1 instance of `KtFilters`

**Review by FlorianWendelborn — CHANGES_REQUESTED** [review]

File: packages/documentation/pages/usage/components/filters.vue (hunk 3 above)
Comment (santiagoballadares) [reply packages/documentation/pages/usage/components/filters.vue:11]: This is the usage example of `KtFilters`. Isn't the user supposed to make sure to provide unique strings for `dataTest` when using multiples instances of KtFilters?

**Review by FlorianWendelborn — APPROVED** [review]

**Pull request merged**